src/_includes/layouts/base.njk

```diff
@@ -14,14 +14,16 @@
     {% if hasBanner %}
     <div class="banner bg-{{ bannerBgColor }}">
       <div class="wrapper flow">
-        <h1>{{ bannerTitle | renderContent('md') | safe }}</h1>
-        {{ bannerBody | renderContent('md') | safe }}
+        <div class="banner__content">
```

Comment:
One thing I realized is that we need to move the `.flow` class onto the new `.banner__content` div so that the text has appropriate vertical spacing.